test/test_coriolis.jl

```diff
@@ -12,15 +12,16 @@ function instantiate_fplane_2(FT)
 end
 
 function instantiate_ntfplane_1(FT)
-    coriolis = NonTraditionalFPlane(FT, fz=π, fy=π)
-    @test coriolis.fz == FT(π)
-    @test coriolis.fy == FT(π)
+    coriolis = GeneralFPlane(FT, coriolis_frequency=1, rotation_axis=[0, cosd(45), sind(45)])
+    @test coriolis.fy == FT(cosd(45))
+    @test coriolis.fz == FT(sind(45))
 end
 
 function instantiate_ntfplane_2(FT)
```

Comment:
since `NontraditionalFPlane` is going to disappear, would we want to change the name of this function appropriately?  Say 'instantiate_general_fplane_2` or something?

Reply:
Also lets definitely not break the convention of using verbose names!

Reply:
Ah, good point. I missed that :)